dist-raw/node-primordials.js

```diff
@@ -1,24 +1,427 @@
-module.exports = {
-  ArrayIsArray: Array.isArray,
-  ArrayPrototypeJoin: (obj, separator) => Array.prototype.join.call(obj, separator),
-  ArrayPrototypeShift: (obj) => Array.prototype.shift.call(obj),
-  ArrayPrototypeForEach: (arr, ...rest) => Array.prototype.forEach.apply(arr, rest),
-  JSONParse: JSON.parse,
-  JSONStringify: JSON.stringify,
-  ObjectFreeze: Object.freeze,
-  ObjectGetOwnPropertyNames: Object.getOwnPropertyNames,
-  ObjectDefineProperty: Object.defineProperty,
-  ObjectPrototypeHasOwnProperty: (obj, prop) => Object.prototype.hasOwnProperty.call(obj, prop),
-  RegExpPrototypeTest: (obj, string) => RegExp.prototype.test.call(obj, string),
-  SafeMap: Map,
-  SafeSet: Set,
-  StringPrototypeEndsWith: (str, ...rest) => String.prototype.endsWith.apply(str, rest),
-  StringPrototypeIncludes: (str, ...rest) => String.prototype.includes.apply(str, rest),
-  StringPrototypeLastIndexOf: (str, ...rest) => String.prototype.lastIndexOf.apply(str, rest),
-  StringPrototypeIndexOf: (str, ...rest) => String.prototype.indexOf.apply(str, rest),
-  StringPrototypeReplace: (str, ...rest) => String.prototype.replace.apply(str, rest),
-  StringPrototypeSlice: (str, ...rest) => String.prototype.slice.apply(str, rest),
-  StringPrototypeSplit: (str, ...rest) => String.prototype.split.apply(str, rest),
-  StringPrototypeStartsWith: (str, ...rest) => String.prototype.startsWith.apply(str, rest),
-  StringPrototypeSubstr: (str, ...rest) => String.prototype.substr.apply(str, rest),
+// copied from https://github.com/nodejs/node/blob/394fdecc9141aa4c6a8ea7f1aecaf5fcaaaf5c72/lib/internal/per_context/primordials.js
```

Comment:
Is this change necessary, or can we stick to the more minimal implementation we had previously?